### PR TITLE
Feature/implement an endpoint to patch artist details

### PR DESCRIPTION
**Pull request by Afaqsoftaxus**
## Jira

- https://softaxus-jira.atlassian.net/browse/CRYP-70

## Changes

- Created Endpoint for updating Artist.

## Testing

-https://softaxus-jira.atlassian.net/browse/CRYP-70?focusedCommentId=10136

### CODE REVIEW

File: src/CryptoAxus.API/Controllers/ArtistController.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
-﻿namespace CryptoAxus.API.Controllers;
+﻿using Microsoft.AspNetCore.Mvc.Formatters;
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.API/Controllers/ArtistController.cs:2]: Why usings have been added here?

File: src/CryptoAxus.API/Controllers/ArtistController.cs [hunk 2]
```diff
@@ -158,6 +160,35 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
...
+    /// <response code="400">Bad Request response with 400 code and information message</response>
+    /// <returns></returns>
+    [HttpPatch("{userWalletAddress:required}", Name = "PatchArtist", Order = 6)]
+    [RequiresParameter(Name = "userWalletAddress", Required = true, Source = OpenApiParameterLocation.Path, Type = typeof(string))]
+    [RequiresParameter(Name = "artistDto", Required = true, Source = OpenApiParameterLocation.Body, Type = typeof(JsonPatchDocument<UpdateArtistDto>))]
+    [SwaggerRequestExample(typeof(PatchArtistRequest), typeof(PatchArtistRequestExample))]
+    [ProducesResponseType(typeof(PatchArtistResponse), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(NotFoundPatchArtistResponse), (int)HttpStatusCode.NotFound)]
+    [ProducesResponseType(typeof(BadRequestPatchArtistResponse), (int)HttpStatusCode.BadRequest)]
+    public async Task<IActionResult> PatchArtist([FromRoute] string userWalletAddress, [FromBody]  JsonPatchDocument<UpdateArtistDto> artist)
```
Comment (junaid-softaxus) [inline src/CryptoAxus.API/Controllers/ArtistController.cs:180]: Extra spacing between `[FromBody]` and `JsonPatchDocument<UpdateArtistDto>`.

File: src/CryptoAxus.API/Controllers/ArtistController.cs [hunk 3]
```diff
@@ -158,6 +160,35 @@ public async Task<IActionResult> PostArtist([FromBody] CreateArtistDto artist)
...
+    {
+           var response = await Mediator.Send(new PatchArtistRequest(userWalletAddress, artist));
```
Comment (junaid-softaxus) [inline src/CryptoAxus.API/Controllers/ArtistController.cs:182]: Alignment issue.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:1]: Incorrect name of the feature folder. We are not performing `PatchArtistWalletAddress` should have been `PatchArtist`.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs [hunk 4]
```diff
@@ -0,0 +1,43 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistUserWalletAddress.Handler;
+public class PatchArtistHandler : BaseHandler<PatchArtistHandler>, IRequestHandler<PatchArtistRequest, PatchArtistResponse>
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:2]: Spacing.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+{
+    private readonly IRepository<ArtistDocument> _repository;
+
+    public PatchArtistHandler(IRepository<ArtistDocument> repository)
+    {
+        _repository = repository;
+    }
+
+    public async Task<PatchArtistResponse> Handle(PatchArtistRequest request,
+                                                          CancellationToken cancellationToken = default)
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:12]: Alignment.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+        ArgumentException.ThrowIfNullOrEmpty(request.UserWalletAddress, nameof(request.UserWalletAddress));
+
+        ArtistDocument artist = await _repository.FindOneAsync(x => x.UserWalletAddress.Equals(request.UserWalletAddress));
+
+        if (artist == null)
+            return new PatchArtistResponse(HttpStatusCode.NotFound, $"Unable to find artist with wallet address: {request.UserWalletAddress}");
+
+        UpdateArtistDto artistDto = new UpdateArtistDto();
+        request.ArtistDto.ApplyTo(artistDto);
+        FilterDefinition<ArtistDocument> filterDefinition =
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:23]: Spacing.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+        UpdateDefinition<ArtistDocument> updateDefinition = Builders<ArtistDocument>.Update
+            .Set(x => x.Username, artistDto.Username ?? artist.Username )
+            .Set(x => x.Email, artistDto.Email?? artist.Email)
+            .Set(x => x.UserWalletAddress, artistDto.UserWalletAddress ?? artist.UserWalletAddress)
+            .Set(x => x.ProfileImageAddress, artistDto.ProfileImageAddress ?? artist.ProfileImageAddress)
+            .Set(x => x.Website, artistDto.Website ?? artist.Website)
+            .Set(x => x.Bio, artistDto.Bio ?? artist.Bio)
+            .Set(x => x.CoverImageAddress, artistDto.CoverImageAddress ?? artist.CoverImageAddress)
+            .Set(x => x.Instagram, artistDto.Instagram ?? artist.Instagram)
+            .Set(x => x.Twitter, artistDto.Twitter ?? artist.Twitter);
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:35]: Where are the fields for `lastModifiedDate`, `lastModifiedBy`?

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+        var updateResult = await _repository.UpdateOneAsync(filterDefinition, updateDefinition);
+
+        return updateResult.ModifiedCount <= 0
+                ? new PatchArtistResponse(HttpStatusCode.BadRequest, "Unable to update the username")
+                : new PatchArtistResponse(HttpStatusCode.NoContent, "Artist username updated successfully");
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:41]: Are we updating the artist's username here? Is this response message correct?

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequest.cs [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+    [JsonProperty("userWalletAddress", Required = Required.Always)]
+    public string UserWalletAddress { get; set; }
+
+    [JsonProperty("artist")]
+
+    public JsonPatchDocument<UpdateArtistDto> ArtistDto { get; set; }
+
+    public PatchArtistRequest(string userWalletAddress, JsonPatchDocument<UpdateArtistDto> artist) => (UserWalletAddress, ArtistDto) = (userWalletAddress, artist);
+
+}
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequest.cs:14]: Alignment and spacing.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs [hunk 10]
```diff
@@ -0,0 +1,18 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistUserWalletAddress.Request;
+
+public class PatchArtistRequestExample : IExamplesProvider<PatchArtistRequest>
+{
+    public PatchArtistRequest GetExamples()
+    {
+        {
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs:7]: Why are these extra brackets added here?

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+            JsonPatchDocument<UpdateArtistDto> patchDocument = new JsonPatchDocument<UpdateArtistDto>();
+            patchDocument.Operations.Add(new Operation<UpdateArtistDto>
+            {
+                op = "replace",
+                path = "/userWalletAddress",
+                value = "test.userWalletAddress"
+            });
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs:14]: Should this example only contain one field? We are not updating the username here?

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponse.cs [hunk 12]
```diff
@@ -0,0 +1,12 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistUserWalletAddress.Response;
+
+public class PatchArtistResponse : BaseResponse<object>
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponse.cs:3]: `BaseResponse<object>` should not be returned here. It should be `BaseResponse<ArtistDto>`

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponseExample.cs [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+    public PatchArtistResponse GetExamples()
+    {
+        return new PatchArtistResponse
+        {
+            ApiException = null,
+            Errors = null,
+            IsSuccessful = true,
+            Links = null,
+            Message = "Record updated successfully",
+            Result = null,
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponseExample.cs:13]: Result field should contain the example values for the `ArtistDto`.

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs:1]: Why this file is a part of this PR?

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs [hunk 14]
```diff
@@ -0,0 +1,52 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistuserAddress.Handler;
```
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:1]: Incorrect namespace.

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+
+    // Act
+    var response = await sut.Handle(request);
+
+    // Assert
+    response.StatusCode.ShouldBe(HttpStatusCode.BadRequest);
+    response.Result.ShouldBeNull();
+    response.IsSuccessful.ShouldBe(false);
+}
+}
```
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:52]: Incorrect alignment and indentation.

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+
+    var request = CreateRequest();
+
+    // Act
+    var response = await sut.Handle(request);
+
+    // Assert
+    response.StatusCode.ShouldBe(HttpStatusCode.NoContent);
+    response.Result.ShouldBeNull();
+}
```
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:19]: Incorrect test case.

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+
+[Fact]
+public async Task When_Artist_Not_Found_By_UserWalletAddress_Expect_Not_Found_Result()
```
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:22]: Test case name should be `When_Artist_Not_Found_Expect_Not_Found_Result`.

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs [hunk 18]
```diff
@@ -0,0 +1,52 @@
...
+
+    // Act
+    var response = await sut.Handle(request);
+
+    // Assert
+    response.StatusCode.ShouldBe(HttpStatusCode.NotFound);
+}
+
+[Fact]
+public async Task When_Artist_Username_Is_Updated_Expect_Bad_Request_Response()
```
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:37]: Incorrect test case name.

File: test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs:1]: Why this file is part of this PR?

**Comment by junaid-softaxus** [comment]
I am marking this pull request for rejection.
@softaxus-shahbaz

File: src/CryptoAxus.API/Controllers/ArtistController.cs (hunk 1 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.API/Controllers/ArtistController.cs:2]: Resolved

File: src/CryptoAxus.API/Controllers/ArtistController.cs (hunk 2 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.API/Controllers/ArtistController.cs:180]: Resolved

File: src/CryptoAxus.API/Controllers/ArtistController.cs (hunk 3 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.API/Controllers/ArtistController.cs:182]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs (hunk 4 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:2]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs (hunk 5 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:12]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs (hunk 6 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:23]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs (hunk 7 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:35]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs (hunk 8 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:41]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequest.cs (hunk 9 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequest.cs:14]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponse.cs (hunk 12 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Response/PatchArtistResponse.cs:3]: Resolved

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs (hunk 14 above)
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:1]: Resolved

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs (hunk 15 above)
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:52]: Resolved

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs (hunk 17 above)
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:22]: Resolved

File: test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs (hunk 18 above)
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PatchArtistWalletAddress/Handler/PatchArtistTests.cs:37]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs (hunk 10 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs:7]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Handler/PatchArtistHandler.cs:1]: Resolved

File: test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs:1]: I have undo the changes in it 
Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs:1]: I have undo the changes 
Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs (hunk 11 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs:14]: Resolved

File: src/CryptoAxus.API/Handlers/GetAllArtistQueryHandler.cs
Comment (junaid-softaxus) [inline src/CryptoAxus.API/Handlers/GetAllArtistQueryHandler.cs:1]: Should not be part of this PR.

File: src/CryptoAxus.Application/Features/Artist/PatchArtist/Handler/PatchArtistHandler.cs [hunk 19]
```diff
@@ -0,0 +1,47 @@
...
+            .Set(x => x.Email, artistDto.Email?? artist.Email)
+            .Set(x => x.UserWalletAddress, artistDto.UserWalletAddress ?? artist.UserWalletAddress)
+            .Set(x => x.ProfileImageAddress, artistDto.ProfileImageAddress ?? artist.ProfileImageAddress)
+            .Set(x => x.Website, artistDto.Website ?? artist.Website)
+            .Set(x => x.Bio, artistDto.Bio ?? artist.Bio)
+            .Set(x => x.CoverImageAddress, artistDto.CoverImageAddress ?? artist.CoverImageAddress)
+            .Set(x => x.Instagram, artistDto.Instagram ?? artist.Instagram)
+            .Set(x => x.Twitter, artistDto.Twitter ?? artist.Twitter)
+            .Set(x => x.LastModifiedBy, artist.LastModifiedBy ?? artist.LastModifiedBy)
+            .Set(x => x.LastModifiedDate, artist.LastModifiedDate ?? artist.LastModifiedDate);
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtist/Handler/PatchArtistHandler.cs:39]: Incorrect implementation. Both the values are same.

File: src/CryptoAxus.Application/Features/Artist/PatchArtist/Request/PatchArtistRequest.cs [hunk 20]
```diff
@@ -0,0 +1,15 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtist.Request;
+
+public class PatchArtistRequest : IRequest<PatchArtistResponse>
+{
+    [JsonProperty("userWalletAddress", Required = Required.Always)]
+    public string UserWalletAddress { get; set; }
+
+    [JsonProperty("artist")]
+
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtist/Request/PatchArtistRequest.cs:9]: Spacing?

Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtist/Request/PatchArtistRequest.cs:9]: Spacing?

File: src/CryptoAxus.Application/Features/Artist/PatchArtist/Response/PatchArtistResponseExample.cs [hunk 21]
```diff
@@ -0,0 +1,17 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtist.Response;
+public class PatchArtistResponseExample : IExamplesProvider<PatchArtistResponse>
```
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtist/Response/PatchArtistResponseExample.cs:2]: Spacing?

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs
Comment (junaid-softaxus) [inline src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs:1]: This file should not be part of this PR.

File: test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs
Comment (junaid-softaxus) [inline test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs:1]: This file should not be part of this PR.

**Review by junaid-softaxus — CHANGES_REQUESTED** [review]
Changes required.

File: src/CryptoAxus.Application/Features/Artist/PatchArtist/Request/PatchArtistRequest.cs (hunk 20 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtist/Request/PatchArtistRequest.cs:9]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtist/Response/PatchArtistResponseExample.cs (hunk 21 above)
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtist/Response/PatchArtistResponseExample.cs:2]: Resolved

File: src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs
Comment (Afaqsoftaxus) [reply src/CryptoAxus.Application/Features/Artist/PatchArtistUsername/Handler/PatchArtistUsernameHandler.cs:1]: This was in first commit by mistake.

File: test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs
Comment (Afaqsoftaxus) [reply test/CryptoAxus.Application.Features.Artist/PostArtist/PostArtistTestsData.cs:1]: This was in first commit by mistake.

File: src/CryptoAxus.API/Handlers/GetAllArtistQueryHandler.cs
Comment (Afaqsoftaxus) [reply src/CryptoAxus.API/Handlers/GetAllArtistQueryHandler.cs:1]: This was in first commit by mistake.

**Review by junaid-softaxus — APPROVED** [review]
